roles/network_plugin/cilium/templates/cilium-config.yml.j2

```diff
@@ -10,32 +10,89 @@ data:
   etcd-config: |-
     ---
     endpoints:
-{% for ip_addr in etcd_access_addresses.split(',') %}
-      - {{ ip_addr }}
-{% endfor %}
-
+      - https://cilium-etcd-client.kube-system.svc:2379
```

Comment:
Am I correct to assume that this is will change storage from cluster etcd to cilium managed etcd?